docs/pages/playground.vue

```diff
@@ -103,4 +151,30 @@
 h1, h2 {
   margin: 20px 0;
 }
+
+.loading-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0.8);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.5em;
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border-width: 0;
+  visibility: hidden;
```

Comment:
Hi @BabyElias, followig-up on our chat. So I played around this page for a bit and I think that this line is causing trouble. When I keep it, my Orca screen reader doesn't announce the updates. After removing it, it does. I think it is caused by

> Using a visibility value of hidden on an element will remove it from the [accessibility tree](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#accessibility_apis). This will cause the element and all its descendant elements to no longer be announced by screen reading technology. ([source](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns))

Reply:
Here's [some guidance on hiding the ARIA live regions](https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-2/#choose-an-appropriate-hiding-technique-if-the-live-region-isn%E2%80%99t-visible):

<img src="https://github.com/learningequality/kolibri-design-system/assets/13509191/d10e0fd4-cda0-435d-9485-a60cab6b0740" width="400" />

I think you don't need to be much worried about it here though since it's just for testing, so whatever technique works for the screen reader will do.

Reply:
Ahh, just removing that makes the NVDA screen reader on my system too announce the loading states. Thank you so much for your time in debugging this! Made the required changes.

Reply:
🙂